apps/platform/next.config.mjs

```diff
@@ -1,14 +1,18 @@
-/** @type {import('next').NextConfig} */
+import createNextIntlPlugin from 'next-intl/plugin';
 import path from 'path';
 
+const withNextIntl = createNextIntlPlugin();
+
+/** @type {import('next').NextConfig} */
 const nextConfig = {
   webpack: (config) => {
     config.resolve.fallback = { fs: false, net: false, tls: false };
     config.resolve.symlinks = true;
+    // Uncomment and adjust these lines if you need to resolve specific modules
     // config.resolve.modules = [
-    // path.resolve('./src'),
-    // path.resolve('../../node_modules'),
-    // path.resolve('../../packages/kit/node_modules')
+    //   path.resolve('./src'),
+    //   path.resolve('../../node_modules'),
+    //   path.resolve('../../packages/kit/node_modules')
```

Comment:
What issues were you having that you had to disable these?

Reply:
I can't remember, looked at file history it seems that I introduce this, maybe I copy pasted these lines tring to solve some problem. Can you also check, if we don't need it I will remove it, thx